test/testLib.py

```diff
@@ -12,76 +12,750 @@
  * GNU General Public License for more details.
 '''
 '''
- * @file commandParser.py
- * @author Dustin Wagner
+ * @file testLib.py
+ * @author Dustin Wagner, Daniel Sacro
  * @date 2021-06-28
 '''
 
-'''This File contains no actual tests. Only a class to send/receive and format results of tests'''
+'''In addition to sending/receiving CSP packets and formatting test results, the class in this 
+file also contains a function that runs the OBC system-wide housekeeping test'''
```

Comment:
This is too much responsibility for this class. It needs to only send/receive and format. It is not a test, it's a facillitator

Reply:
I'm not sure what you're suggesting. Could you elaborate?

Reply:
Poke

Reply:
This is covered by my task to refactor the tests

Reply:
@tganley